azkustoingest/managed.go

```diff
@@ -46,6 +54,13 @@ func NewManaged(client QueryClient, db, table string, options ...Option) (*Manag
 	}, nil
 }
 
+func newManagedFromClients(queued *Ingestion, streaming *Streaming) *Managed {
```

Comment:
Why preserve this options?

Reply:
for tests that need mocks, again - it's not public